### PR TITLE
Feature/time

### PR DESCRIPTION
**Pull request by sfriedlizuehlke**
Add Logs marks PERF_TIME

### CODE REVIEW

File: polytope_server/common/staging/s3_staging.py [hunk 1]
```diff
@@ -126,9 +129,19 @@ def create(self, name, data, content_type):
             self.client._complete_multipart_upload(self.bucket, name, upload_id, parts)
         except Exception:
             self.client._remove_incomplete_upload(self.bucket, name, upload_id)
+            end = timeit.default_timer()
+            delta = timedelta(seconds=(end - start)).total_seconds()
```
Comment (cfkanesan) [inline polytope_server/common/staging/s3_staging.py]: The timedelta object is not really needed in this situation.

File: polytope_server/common/staging/s3_staging.py [hunk 2]
```diff
@@ -126,9 +129,19 @@ def create(self, name, data, content_type):
...
+            logging.info(
+                f"PERF_TIME fdb+s3 request_id, seconds, size, MB/s: {name},{delta:.4f},{total_size},{(total_size/1024/1024/delta):.2f}"
```
Comment (cfkanesan) [inline polytope_server/common/staging/s3_staging.py]:
```suggestion
                f"PERF_TIME fdb+s3 request_id, elapsed [s], size [bytes], throughput [MiB/s]: {name},{delta:.4f},{total_size},{(total_size/1024/1024/delta):.2f}"
```

File: polytope_server/common/staging/s3_staging.py [hunk 3]
```diff
@@ -71,6 +73,7 @@ def __init__(self, config):
         )
 
     def create(self, name, data, content_type):
+        start = timeit.default_timer()
```
Comment (cfkanesan) [inline polytope_server/common/staging/s3_staging.py:75]: You could use a decorator to wrap the function and avoid having 2 exit points. Alternatively, you could move the exit logic into a finally clause.

File: polytope_server/common/staging/s3_staging.py (hunk 1 above)
Comment (sfriedlizuehlke) [reply polytope_server/common/staging/s3_staging.py]: I removed the timedelta

File: polytope_server/common/staging/s3_staging.py (hunk 3 above)
Comment (sfriedlizuehlke) [reply polytope_server/common/staging/s3_staging.py:75]: For now I got rid of the log in the exception case as this should work for our current use case.

**Review by cfkanesan — APPROVED** [review]

**Pull request merged**